reflex/components/radix/primitives/accordion.py

```diff
@@ -36,6 +35,7 @@ def get_theme_accordion_root(variant: Var[str], color_scheme: Var[str]) -> BaseV
     Returns:
         The theme for the accordion root component.
     """
+    _var_datas: list[VarData] = []
```

Comment:
extracting var_datas from match cases because the `var_data` is lost when a dict is converted to emotion style dict via `format_as_emotion`